addons/mail/views/discuss_category_views.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+
+        <record id="discuss_category_view_tree" model="ir.ui.view">
+            <field name="name">discuss.category.list</field>
+            <field name="model">discuss.category</field>
+            <field name="arch" type="xml">
+                <list string="Discuss Categories" editable="bottom" sample="1">
+                    <field name="sequence" widget="handle"/>
+                    <field name="name" placeholder="e.g. Service"/>
+                    <field name="channel_ids" widget="many2many_tags"/>
+                </list>
+            </field>
+        </record>
+
+        <record id="discuss_category_view_form" model="ir.ui.view">
+            <field name="name">discuss.category.form</field>
+            <field name="model">discuss.category</field>
+            <field name="arch" type="xml">
+                <form string="Discuss Categories">
+                    <sheet>
+                        <group>
+                            <field name="name" placeholder="e.g. Service"/>
+                            <field name="channel_ids" widget="many2many_tags"/>
```

Comment:
This is not the right widget, the field is not a many2many.

We shouldn't be able to add the same channel in different categories.

<img width="312" height="169" alt="image" src="https://github.com/user-attachments/assets/5fc7f0e3-9ccf-47ae-85af-05bda3cfd169" />

Reply:
If there is no better widget we should probably make it readonly then (or create a custom one), because adding the same channel in 2 categories should be not allowed.